libs/react/src/lib/form/fieldset/fieldset.stories.mdx

```diff
@@ -0,0 +1,76 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs'
+import { Fieldset } from './fieldset'
+import { Input } from '../input/input' 
+import { RadioButton } from '../radioButton/radioButton';
+import { RadioGroup } from '../radioButton/radioGroup';
+import { Option, Select } from '../../select/select'
+
+export const FieldsetTemplate = ({ ...props }) => <Fieldset {...props} />
+
+<Meta title="Components/Form/Fieldset" component={Fieldset} />
+
+# Fieldset
+
+Fieldset with padding, margin and background used to group inputs.
+
+## Basic
+
+<Canvas>
+  <Story
+    name="Fieldset"
+    parameters={{ componentIds: ['component-fieldset'] }}
+    args={{ children: 'Content' }}
+  >
+    {FieldsetTemplate.bind({})}
+  </Story>
+</Canvas>
+
+## Margin
+
+<Canvas>
+  <div>
+    <Fieldset>
+      <Input label="Input 1" />
+    </Fieldset>
+    <Fieldset marginBottom>
+      <Input label="Input 1" />
+    </Fieldset>
+    <Fieldset>
+      <Input label="Input 1" />
+    </Fieldset>
+  </div>
+</Canvas>
+
+## With legend
```

Comment:
It's possible to add a legend, but I don't think it is part of the design system. But still wanted to show how it would look like.